ublox/benches/packet_benchmark.rs

```diff
@@ -48,20 +49,24 @@ pub fn criterion_benchmark(c: &mut Criterion) {
     for chunk in &[99, 100, 101, 256, 512, 1000, 1024] {
         c.bench_function(&format!("vec_parse_pos_{}", chunk), |b| {
             b.iter(|| {
-                let data = std::include_bytes!("pos.ubx");
-                let parser = Parser::default();
-                assert_eq!(parse_all(parser, data, *chunk), 2801);
+                // TODO: requires pos.ubx file
+                // let data = std::include_bytes!("pos.ubx");
```

Comment:
Missing `pos.ubx` file

Reply:
I presume this modification is there to make clippy happy ?

Reply:
If everyone is happy with these changes, benchmarking changes can be reverted after merging -- Will leave this PR open for a bit

Reply:
that's alright for me, @lkolbly personal computer is probably the only place where this was ever executed.
This can probably benefit some rework in the future